pkg/networkverifier/networkverifier.go

```diff
@@ -76,7 +76,6 @@ func InitializeValidateEgressInput(cluster *v1.Cluster, clusterDeployment *hivev
 	}
 
 	return &verifier.ValidateEgressInput{
-		Timeout:      5 * time.Second,
```

Comment:
The upstream default is 5 sec as well, we ideally don't want to manually override this and inherit it from upstream directly.